components/atoms/Alert.js

```diff
@@ -10,12 +11,24 @@ import { faCircleInfo } from "@fortawesome/free-solid-svg-icons";
 export function Alert(props) {
   return (
     <div className="layout-container">
-      <div className="relative border-l-6 border-[#269ABC] min-h-40px my-10">
+      <div
+        className={`relative border-l-6 min-h-40px my-10 ${
+          props.triangle ? "border-custom-orange-warning" : "border=[#269ABC]"
```

Comment:
```suggestion
          props.triangle ? "border-custom-orange-warning" : "border-[#269ABC]"
```

Typo here, info alert border color wasn't showing up properly